configuration/rules-dsl.md

```diff
@@ -92,6 +96,8 @@ then
 end
 ```
 
+Note that the rule name must match the filename of the rule, minus the extension ".rules"
```

Comment:
Hi Thom,
Thank you very much for making me feel welcome.  Some communities are not as open to input.

I have no specific area that I would want to work on.  I always keep my own installation notes when I set up a system, but I think I could help the community by simply noting any changes or new things I discover that are not in the documentation by contributing something to the documentation.  I wrote a Computer and Networking column in a trade magazine for ten years, so writing technical documentation and articles is one of the things I have some experience doing.
For right now, I am struggling through the documentation and many different places to get rules working.  So that may be my short-term focus.  But as I have time, I am happy to take a pass through the rest of the documentation and contribute where I can.  You guys have done an amazing job, overall, but there are some places where it is possible that a native English speaker might have said things slightly differently.  I definitely do not want to upset anyone with a lot of suggestions that delete a coma or change only a word or two, but these small changes can really make a difference.  Do you know the book with the title, Eats Shoots and Leaves?  If you write the sentence, "Eats shoots and leaves", you could be talking about a Panda.  But if you write, "Eats, shoots, and leaves", then you could be talking about one of the all-too frequent and sad situations we read in the news these days.
Regarding your first comment, yes, absolutely following #189 would be the right thing to do. So now can you please teach me about GitHub?  I am using Git Desktop.  What is the correct way to make this additional change when I have already submitted a PR?
Regarding your comment about file names, I have had some really buggy behavior on rules files in my particular situation which has made troubleshooting it a nightmare.  For example, I get a note in my log file saying that a new rules file has been read, but I still have to restart the application in order to get openHAB2 to recognize the new file.  I may have just gotten confused about the rule name - not sure.  But in any case, I have just tested it now that I moved to the apt "Tested" branch instead of "Stable", and you are correct, so that change is invalid and should not be incorporated.